optimum/habana/transformers/modeling_utils.py

```diff
@@ -261,6 +267,19 @@ def adapt_transformers_to_gaudi():
     transformers.models.mpt.modeling_mpt.MptAttention.forward = gaudi_mpt_attention_forward
     transformers.models.mpt.modeling_mpt.MptBlock.forward = gaudi_mpt_block_forward
 
+    # TODO: revisit this when switching to Transformers v4.33
+    # see https://github.com/huggingface/transformers/pull/25394
+    # Hack for running T5 with DeepSpeed Zero-3
+    # Dropout kernel improvement for Flan-T5
```

Comment:
This should not be there anymore, see main: https://github.com/huggingface/optimum-habana/blob/3aa707e386f9981e7b0ff5da7c52c87412b9a1e8/optimum/habana/transformers/modeling_utils.py#L257
I think `gaudi_t5_layernorm_forward` got lost at some point when solving merge conflicts.

Reply:
https://github.com/huggingface/optimum-habana/blob/2b3323efbbfa41caab28f7b69a9ff8db15539a2b/optimum/habana/transformers/modeling_utils.py#L262

gaudi_t5_layernorm_forward is there

Reply:
My bad I read too quickly